examples/compound-assignment-operators/compound_assignment_operators.description

```diff
@@ -1,2 +1,2 @@
 // Compound assignment operators can be used to simplify the code and increase readability. These operators can only be
-// used as statements (i.e. cannot be used as expressions).
```

Comment:
Extra comma added here, any reason?

Reply:
This was as suggested by Sam in https://github.com/ballerina-guides/messaging-with-jms-queues/pull/13#discussion_r199154292,

Reply:
We need to use a comma after abbreviations such as:
- i.e.,
- e.g.,